app/src/test/java/co/nimblehq/testutil/MockSchedulersProvider.kt

```diff
@@ -1,9 +1,9 @@
 package co.nimblehq.testutil
 
-import co.nimblehq.data.lib.schedulers.SchedulersProvider
+import co.nimblehq.domain.schedulers.BaseSchedulerProvider
 import io.reactivex.schedulers.Schedulers
 
-object MockSchedulersProvider : SchedulersProvider {
+object MockSchedulersProvider : BaseSchedulerProvider {
```

Comment:
What about `TrampolineSchedulerProvider` Can we use it instead `MockSchedulersProvider` 🤔 Correct me if I miss somthing 🙏

Reply:
You are right, they are the same 🙏  Will update